compress_jointcal_cfht_test_data.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+"""Zero out the imageHDUs in the calexps, and then gzip the files."""
+from __future__ import absolute_import, division, print_function
+
+from astropy.io import fits
+import glob
+import subprocess
+
+files = glob.glob('cfht/calexp/06AL01/D3/2006-0*/r/calexp*.fits')
+for f in files:
+    data = fits.open(f)
+    print('processing:', f)
+    for i in range(len(data)):
+        if isinstance(data[i], fits.ImageHDU):
+            data[i].data[:] = 0
```

Comment:
This (zeroing and compressing) is a very nice trick for reducing the data size.  But it does mean that the user can't look at the images to see, e.g., why a particular source doesn't get a good fit.